src/states/abstract/State.js

```diff
@@ -106,8 +106,8 @@ export default class State {
       this.onAdTimeout(player);
     } else if (type === 'ads-ad-started') {
       this.onAdStarted(player);
-    } else if (type === 'contentupdate') {
-      this.onContentUpdate(player);
+    } else if (type === 'contentchanged') {
```

Comment:
just for confirmation: contentupdate will no longer be used for state transitions?

Reply:
Correct